packages/dd-trace/src/profiling/profiler.js

```diff
@@ -137,6 +147,25 @@ class Profiler extends EventEmitter {
     }
   }
 
+  _onSpanFinish (span) {
+    const tags = span.context()._tags
+    if (tags[SPAN_TYPE] !== WEB) return
+
+    const endpointName = tags[RESOURCE_NAME] || [
+      tags[HTTP_METHOD],
+      tags[HTTP_ROUTE]
+    ].filter(v => v).join(' ')
+    if (!endpointName) return
+
+    let counter = this.endpointCounts.get(endpointName)
+    if (counter === undefined) {
+      counter = { count: 1 }
```

Comment:
The use of an object with a `count` field as value instead of directly a number is to allow for extensibility ?

Reply:
No, I just didn't want to always have to call `set` after a `get` on the Map.
```
const count = this.endpointCounts.get(endpointName) || 0
this.endpointCounts.set(endpointName, count++)
```
Of course, if I used an object instead of a Map, I wouldn't have to do this.

Reply:
Ok, I get it ;)
Get returns you a reference that you can update contrary to a number.